filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -306,19 +306,23 @@ The default setting is 10s.
 [[scan-sort]]
 ===== scan.sort
 
-Specifies if files should be harvested in order and how to determine the order. Possible values are modtime, filename and none. To sort by file modification time use modtime otherwise use filename.
+[]experimental
 
-If you specify a value other than none for this setting you can determine whether to use ascending or descending order using `scan.order`.
+Specifies if files should be harvested in order and how to determine the order. Possible values are `modtime`, `filename`. Leaving the option empty means its disabled. To sort by file modification time use modtime otherwise use filename.
 
-The default setting is none.
+If you specify a value for this setting you can determine whether to use ascending or descending order using `scan.order`.
```

Comment:
Change to say: If you specify a value for this setting, you can configure whether files are scanned in ascending or descending order by using `scan.order`.